Add secret manager description (#165)
MinBZK/ai-validation: docs/way-of-working/onboarding/accounts.md

- [Team Collaboration Space](https://www.samenwerkruimten.nl/teamsites/ai-validatie-team)
- [Sprint Board](https://github.com/orgs/MinBZK/projects/7)
- [Webex Room](https://rijksvideo.webex.com/rijksvideo-du/j.php?MTID=mefba1dbb67959de6461040f02aadf353)

## Secrets

> We use Hashicorp [Vault](https://vault.apps.digilab.network/) secrets manager for team secrets. You can login with a
> Github Personal access token. The token needs organization read permissions and you need to be added to the Github team
> to be able to access the vault.

Reviewer: ```suggestion
We use [HashiCorp Vault](https://vault.apps.digilab.network/) secrets manager for team secrets. You can login with a
[GitHub Personal access token](https://github.com/settings/tokens). The token needs organization read
permissions (`read:org`), and you should be part of our GitHub team to access the vault.
```

@berrydenhartog I followed the instructions. Because I used Google to find the GitHub Tokens setting page, I thought it would be handy to add the link in these instructions as well :)
